app/models/user.rb

```diff
@@ -1,6 +1,8 @@
 class User < ApplicationRecord
   has_many :stockrooms, dependent: :destroy
   has_many :items, dependent: :destroy
+  has_many :user_logs, dependent: :destroy
+  has_many :logs, through: :user_logs
```

Comment:
this is the association also that will get the current user logs which is `current_user.logs`. It traverse on the user_logs table(you will see it on the migration)